src/main/java/org/zalando/nakadi/domain/SubscriptionBase.java

```diff
@@ -29,6 +29,7 @@
     private Set<String> eventTypes;
 
     @NotNull
+    @Size(min = 1, message = "must contain at least one character")
```

Comment:
Well, in general yes, but we don't really validate the max size of properties anywhere except the event-type name. So if we decide to do that we should add that for all fields and as a separate task. Doing it right now for this only field will not bring much value as for me.